source/administration/important-upgrade-notes.rst

```diff
@@ -2,13 +2,25 @@ Important Upgrade Notes
-   - The ``platform`` binary and “--platform” flag will be deprecated in a future release. If you are using the “--platform” flag or are using the ``platform`` binary directly to run the Mattermost server application via a systemd file or custom script, you will be required to use only the ``mattermost`` binary.
 
 +----------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | If you’re upgrading from a version earlier than... | Then...                                                                                                                                                          |
 +====================================================+==================================================================================================================================================================+
+| v5.37.0                                            | The ``platform`` binary and “--platform” flag will be deprecated in a future release. If you are using the “--platform” flag or are using the ``platform`` binary|
+|                                                    | directly to run the Mattermost server application via a systemd file or custom script, you will be required to use only the ``mattermost`` binary.               |
+|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                                    | `Collapsed Reply Threads <https://mattermost.com/blog/collapsed-reply-threads-beta/>`_ are available as beta in Mattermost Server                                |
+|                                                    | v5.37 and later. It’s expected that you may experience bugs as we stabilize the feature. In particular, please be aware of                                       |
+|                                                    | `the known issues documented here <https://docs.mattermost.com/help/messaging/organizing-conversations.html#known-issues>`_.                                     |
+|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                                    | v5.37 adds support for emoji standard v13.0. If you have added a custom emoji in the past that uses one of the new system names, then it is going to get         |
+|                                                    | overwritten by the system emoji. The workaround is to change the custom emoji name.                                                                              |
```

Comment:
maybe mention that the emoji picker now supports selecting a default skin tone?

Reply:
The changelog draft mentions it, I think https://github.com/mattermost/docs/pull/4702/files

Reply:
ah! great then :) disregard my comment